### PR TITLE
Add responsiveness for mobile users

### PR DESCRIPTION
**Pull request by tarynmartin**
#### What’s this PR do?
- Add a media query and breakpoint for mobile users and some tweaks to the start activity button.
- Add input type of number to seconds input area in HTML
- Add div for start button on HTML
- Added layout for start activity button and got rid of left border bar in mobile mode.
- Added container for title and moved layout and styling to the container in CSS
#### Where should the reviewer start?
- Start at the media query, then compare to the base CSS code.
#### How should this be manually tested?
- In the browser
#### Questions: 
- Not sure how to align text & icons within a button?
- How to get the buttons to the correct sizes compared to the comp?
- Getting rid of space around title bar?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,10 +38,12 @@ <h1 class="left-title">New Activity</h1>
               </div>
               <div class="right-input">
                 <label>Seconds</label>
-                <input>
+                <input type="number">
               </div>
             </form>
-            <button class="start-btn">START ACTIVITY</button>
+            <div class="btn-start">
```
Comment (kip-west) [inline index.html:46]: I was wondering if we'd need another div/section here!! Did this make responsive-ness easier for you? I was struggling to get this button in it's right position earlier...

File: index.html [hunk 2]
```diff
@@ -38,10 +38,12 @@ <h1 class="left-title">New Activity</h1>
               </div>
               <div class="right-input">
                 <label>Seconds</label>
-                <input>
+                <input type="number">
```
Comment (kip-west) [inline index.html:43]: Great catch!! I totally forgot to get this one...

File: styles.css [hunk 3]
```diff
@@ -107,3 +107,71 @@ input {
...
+    text-align: center;
+    justify-content: space-around;
+  }
+
+  .study-btn,
+  .meditate-btn,
+  .exercise-btn {
+    display: grid;
+    margin: 0 20%;
+    margin-bottom: 2em;
```
Comment (kip-west) [inline styles.css:159]: To get the button text & icons centered, I think we'll be looking at align-items or justify-items in this btn styling.

**Review by kip-west — APPROVED** [review]
This all looks great! The mobile version looks spot-on with the comp. Nice work!! Let's hammer out these button issues tomorrow!

**Review by jobbotrock — APPROVED** [review]
Looks good!

**Pull request merged**